aliases/README.md

```diff
@@ -1 +1,42 @@
-Placeholder for Aliases
\ No newline at end of file
+# Aliases
+The files contained here can be added to your RC files using a variation of the import/include commands.
+We have set up each file to have each alias disabled by default. Unfortunately, this would require you to read through the file to decide what aliases you would like to enable. This is the best way not to interupt your workflow by accidentally overwriting any possible shared alias names.
+
+### Getting started
+#### Setting up your git config file
+1. Read through the `.gitconfig-luitk` file and decide which aliases you would like to enable.
+	- Aliases can be enabled by uncommenting the alias(removing the `#` preceeding the alias)
```

Comment:
"Be sure not to uncomment the actual comments left to help explain what the alias does."